web/templates/settings/user_list_admin.hbs

```diff
@@ -5,7 +5,10 @@
     <div class="settings_panel_list_header">
         <h3>{{t "Users"}}</h3>
         <div class="alert-notification" id="user-field-status"></div>
-        <input type="text" class="search filter_text_input" placeholder="{{t 'Filter users' }}" aria-label="{{t 'Filter users' }}"/>
+        <div class="user_filters">
```

Comment:
Small naming comment: We've been moving towards a convention of preferring to add `-s`  over `_`s in CSS class names; but not a high priority worth reworking anything over.